packages/react/src/formElements/MdMultiSelect.tsx

```diff
@@ -28,8 +28,7 @@ export interface MdMultiSelectProps {
   errorText?: string;
   showChips?: boolean;
   closeOnSelect?: boolean;
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  id?: any;
+  id?: string | number | null | undefined;
```

Comment:
I don't like this but I want to avoid breaking people's builds. And this type for ids is unfortunately common in md-components.